src/covid_model_seiir_pipeline/forecasting/task/beta_forecast.py

```diff
@@ -59,6 +59,11 @@ def run_beta_forecast(draw_id: int, forecast_version: str, scenario_name: str):
     # We'll need this to compute deaths and to splice with the forecasts.
     infection_data = data_interface.load_infection_data(draw_id)
 
+    if ((1 < thetas) | thetas < -1).any():
+        raise ValueError('Theta must be between -1 and 1.')
+    if (beta_params['sigma'] - thetas).any():
```

Comment:
The error message implies this should be `((beta_params['sigma']) - thetas >= 1).any()`. What am I missing?

Reply:
Nothing.  good catch.